pkgs/development/python-modules/aiosmb/default.nix

```diff
@@ -16,14 +16,14 @@
 
 buildPythonPackage rec {
   pname = "aiosmb";
-  version = "0.4.7";
+  version = "0.4.8";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-ze8x0vFGnPAIQQicuJxAcBVEeeKOGUHvepRTO4Ejx+g=";
+    hash = "sha256-siV2cr8YuU4kWRNHGlV29RjeHxCk6mcbK7J15LZguls=";
   };
 
   propagatedBuildInputs = [
```

Comment:
I guess we need to add cryptography to propagatedBuildInputs.
https://github.com/skelsec/aiosmb/blob/7d50c87bd62920284f0b712aa3e5b1df9539e752/setup.py#L52

Reply:
Added, thanks for checking.